scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -1195,38 +1301,79 @@ private void build() {
         return ret;
     }
 
+    private void createJobVariable(Map<String, JobVariable> ret, String name, String value, String model) {
+        if (name != null && value != null) {
+            if (name.matches("[A-Za-z0-9._]+")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)